config/webpack.dev.js

```diff
@@ -16,7 +16,7 @@ module.exports = env => ({
     './index.js',
   ],
 
-  devtool: 'cheap-module-source-map',
+  devtool: 'source-map',
```

Comment:
My understanding is source map is really slow. How about `cheap-module-eval-source-map`